src/app/components/Table/index.tsx

```diff
@@ -112,18 +114,23 @@ export const Table: FC<TableProps> = ({
         <MuiTable aria-label={name}>
           <TableHead>
             <TableRow>
-              {columns.map((column, index) => (
-                <TableCell
-                  key={column.key}
-                  align={column.align}
-                  sx={{
-                    width: column.width || 'auto',
-                    ...(stickyColumn && !index && isMobile ? stickyColumnStyles : {}),
-                  }}
-                >
-                  {column.content}
-                </TableCell>
-              ))}
+              {columns.map((column, index) => {
+                if (column.hide) {
+                  return null
```

Comment:
Note: if first column is hidden then stickyColumn doesn't work

Reply:
we can remove `stickyColumn` it is not used anywhere